hasher-matcher-actioner/hmalib/common/configs/tx_collab_config.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+DynamoDB backed CollaborationConfig objects.
+
+This may become a CollaborationConfigStoreBase, but since none of the pytx
+interfaces require it, keeping it as just another class for now.
+"""
+
+import typing as t
+import itertools
+
+from threatexchange.exchanges.collab_config import CollaborationConfigBase
+from threatexchange.exchanges.signal_exchange_api import SignalExchangeAPI
+
+from hmalib.common.config import HMAConfig, create_config
+
+
+def make_collab_config_subclass(
+    cls: t.Type[CollaborationConfigBase],
+) -> t.Type[HMAConfig]:
+    """
+    Generate an HMAConfig subclass on-the-fly.
+
+    Since supported APIs are dynamic, we need to allow any
+    CollaborationConfigBase to be stored as an HMAConfig object.
+    """
+    return type(f"_Dynamic{cls.__name__}_HMAConfig", (cls, HMAConfig), {})
+
+
+class CollabConfig(HMAConfig):
+    """
+    Stores and retrives CollaborationConfigBase objects as HMAConfig objects. To
+    allow for extending as CollaborationConfigBase get more complex, only known
+    and common attributes are DDB Attributes. Everything else is dumped after
+    serializing.
+    """
+
+    # The subclass of CollaborationConfigBase that we are representing
+    concrete_class: str
+
+    # Attributes in CollaborationConfigBase
+    collab_config_name: str
+    collab_config_api: str
+    enabled: bool
```

Comment:
Isn't this duplicate of values in collabconfig? Some of them are redefinitions (name => collab_config_name) and some of these will shadow (enabled)